test/backend/controllers/integration_analytics.dont_run.js

```diff
@@ -0,0 +1,126 @@
+const expect = require('chai').expect;
+const request = require('supertest');
+const Teacher = require('../../../backend/models/teacher');
+const Student = require('../../../backend/models/student');
+const Analytic = require('../../../backend/models/analytic');
+const Focus_Item = require('../../../backend/models/focus_item');
+const Program = require('../../../backend/models/program');
+const ingest = require('../ingest');
+const app = require('../../../server');
+
+
+describe('Tests analytics controller with fake database', function() {
+    var ingest_results = [];
+    before(function(done) {
+        this.timeout(40000);
+        Analytic.deleteMany({}).then(function() {
+            Teacher.deleteMany({}).then(function() {
+                Student.deleteMany({}).then(function() {
+                    Focus_Item.deleteMany({}).then(function() {
+                        Program.deleteMany({}).then(function() {
+                            // ingest.ingest(15, 4000, 10, done);
+                            // console.log("Starting ingest")
+                            ingest.ingest_custom().then(function(results) {
+                                ingest_results = results;
+                                // console.log(results)
+                                done();
+                            });
+                            // console.log(results)
+                        });
+                    });
+                });
+            });
+        });
+    });
+
+    after(function(done) {
+        Analytic.deleteMany({}).then(function() {
+            Teacher.deleteMany({}).then(function() {
+                Student.deleteMany({}).then(function() {
+                    Focus_Item.deleteMany({}).then(function() {
+                        Program.deleteMany({}).then(function() {
```

Comment:
Hey the file name for this one is intentionally named so it won't run. I just figured we'd have it in the repo because its the only way I've been able to test my code

Reply:
Oooh you're right! I didn't even notice that